frontend/packages/erd-web/app/globals.css

```diff
@@ -0,0 +1,21 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+:root {
+  --background: #ffffff;
+  --foreground: #171717;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background: #0a0a0a;
+    --foreground: #ededed;
+  }
+}
+
+body {
+  color: var(--foreground);
+  background: var(--background);
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
These looked good to delete.

```suggestion
```

Reply:
Deleted unused files, thanks!🙏
369a929bad83ef2fcb60bb01044e882c120002f8